core/vm/actor/builtin/v0/storage_power/storage_power_actor_state.cpp

```diff
@@ -21,13 +26,15 @@ namespace fc::vm::actor::builtin::v0::storage_power {
         .this_epoch_raw_power = {},
         .this_epoch_qa_power = {},
         .this_epoch_pledge = {},
-        .this_epoch_qa_power_smoothed = {},
+        .this_epoch_qa_power_smoothed =
+            {.position = kInitialQAPowerEstimatePosition << kPrecision,
+             .velocity = kInitialQAPowerEstimateVelocity << kPrecision},
         .miner_count = 0,
         .num_miners_meeting_min_power = {},
-        .cron_event_queue = {},
+        .cron_event_queue = {ipld},
```

Comment:
May be omited, because following `ipld->load(state)` will fill it

Reply:
ok